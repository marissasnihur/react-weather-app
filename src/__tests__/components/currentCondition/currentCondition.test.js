import React from 'react';
import ReactDOM from 'react-dom';

import CurrentCondition from '../../../components/CurrentCondition/CurrentCondition';

describe('CurrentCondition component', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CurrentCondition />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should create a <div> tag', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CurrentCondition>Sunny</CurrentCondition>, div);
    expect(div.firstChild.nodeName).toBe('DIV');
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match snapshot', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CurrentCondition>Edmonton</CurrentCondition>, div);
    expect(div).toMatchSnapshot();
    ReactDOM.unmountComponentAtNode(div);
  });
});
