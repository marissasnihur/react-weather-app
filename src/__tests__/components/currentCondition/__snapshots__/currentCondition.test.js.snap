// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentCondition component should match snapshot 1`] = `
<div>
  <div
    class="currentCondition"
  >
    Edmonton
  </div>
</div>
`;
